web/client/plugins/FeatureEditor.jsx

```diff
@@ -81,6 +82,7 @@ const Dock = connect(createSelector(
   * @prop {boolean} cfg.showFilteredObject default false. Displays spatial filter selection area when true
   * @prop {boolean} cfg.showTimeSync default false. Shows the button to enable time sync
   * @prop {boolean} cfg.timeSync default false. If true, the timeSync is active by default.
+  * @prop {boolean} cfg.isSyncActive default false. If true, the wms sync tool will be active by default.
```

Comment:
The code seems good. IMO The only strange thing is the name of the configuration prop `isSyncActive`.
I don't know but maybe it could be more explict `enableMapSync` or `mapSync` to avoid confusion with the time sync configuration.

Reply:
sure, i will rename it to enableMapSync

Reply:
as agreed i have changed it to enableMapFilterSync